<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="apitest" duration-ms="5173" started-at="2019-09-17T08:43:30Z" finished-at="2019-09-17T08:43:35Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="5173" started-at="2019-09-17T08:43:30Z" finished-at="2019-09-17T08:43:35Z">
      <class name="com.bitnei.apitest.testcases.datasend.TaskAdd">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.bitnei.apitest.testcases.datasend.TaskAdd@40f08448]" name="setUp" is-config="true" duration-ms="1387" started-at="2019-09-17T08:43:30Z" finished-at="2019-09-17T08:43:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[http://10.10.26.3:81]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="TaskAddData(java.lang.String)[pri:0, instance:com.bitnei.apitest.testcases.datasend.TaskAdd@40f08448]" name="TaskAddData" duration-ms="3749" started-at="2019-09-17T08:43:32Z" description="数据转发平台新增任务" data-provider="dataprovider1" finished-at="2019-09-17T08:43:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{"status":200,"errorMsg":null,"data":null,"error":false}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [{}] but found [{"status":{"currentValue is ":400,"expectedValue is":200},"error":{"currentValue is ":true,"expectedValue is":false}}]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [{}] but found [{"status":{"currentValue is ":400,"expectedValue is":200},"error":{"currentValue is ":true,"expectedValue is":false}}]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.bitnei.apitest.testcases.datasend.TaskAdd.TaskAddData(TaskAdd.java:99)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TaskAddData -->
      </class> <!-- com.bitnei.apitest.testcases.datasend.TaskAdd -->
    </test> <!-- Default test -->
  </suite> <!-- apitest -->
</testng-results>
